### PR TITLE
adapt license files + format about:license (fix #11655)

### LINKED ISSUE #11655: Missing date and name of the copyright owner in application license
I believe that the [yyyy] [name of copyright owner] segment is meant to be replaced (maybe by CI, to keep the date updated)

### PR DESCRIPTION
**Pull request by moving-bits**
## Description
- adapts license files as described in #11655
- applies Markdown formatting to about:license page for better readability
- and, as I came along while working on those issues, I removed references to our old IRC channel as well

### CODE REVIEW

**Comment by SammysHP** [comment]
I don't think it will work this way. The license file can stay as it was, maybe just the appendix removed. But every source file requires the header.

I suggest to add the template from the appendix to the readme and at the top of the license page in the app.

**Comment by moving-bits** [comment]
Thanks @SammysHP for reviewing. I adapted the PR accordingly:
- removed appendix from license files
- added info about c:geo team etc. at the beginning of about: contributors page
- added info about c:geo team etc. in our README
- and BTW removed the link to the old Freenode chat

**Comment by bekuno** [comment]
With https://github.com/cgeo/cgeo/commit/e05ffa94276a7938e85bb5b5b5e2b1229f186770 the file https://github.com/cgeo/cgeo/blob/e05ffa94276a7938e85bb5b5b5e2b1229f186770/NOTICE was created to cover the license information in c:geo. Now this will be added to the README. 
Can we remove the NOTICE (I did not find where it is used in c:geo or indirect used by GitHub)? If not, then the year should renewed.

**Comment by moving-bits** [comment]
Ok, I've updated the PR to delete the `NOTICE` file

**Comment by bekuno** [comment]
The `NOTICE` file is only necessary for ASF projects, see e.g. https://github.com/github/choosealicense.com/issues/701 . So it is fine to remove it.

**Review by bekuno — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -134,4 +137,3 @@ c:geo is distributed under the [Apache License, Version 2.0](https://www.apache.
 - Facebook: https://www.facebook.com/android.geocaching
 - Google Play: https://play.google.com/store/apps/details?id=cgeo.geocaching
 - Live status: https://status.cgeo.org/
-- Developer chat: [#cgeo on freenode.net](https://webchat.freenode.net/#cgeo)
```
Comment (fm-sys) [inline README.md:137]: BTW, the freenode chat is mentioned in line 13 of this file as well...

Comment (moving-bits) [reply README.md:137]: removed as well, thanks for the hint

**Comment by moving-bits** [comment]
retest this please

**Pull request merged**